### PR TITLE
Cleanup orchestrator proto

### PR DESCRIPTION
**Pull request by JoeZijunZhou**
- Removed unused fields in orchestrator proto
- Update all files according to the proto cleanup

### CODE REVIEW

File: jetstream/core/proto/jetstream.proto [hunk 1]
```diff
@@ -26,9 +26,6 @@ service Orchestrator {
 }
 
 message DecodeRequest {
-  // Where to load any pre-existing kv cache from.
-  string session_cache = 1;
```
Comment (FanhaiLu1) [inline jetstream/core/proto/jetstream.proto:30]: In future, we will have prefill cache logic. Is this filed related with prefill cache?

Comment (JoeZijunZhou) [reply jetstream/core/proto/jetstream.proto:30]: This is for history cache originally. We could add prefill cache fields when we implement it.

Comment (FanhaiLu1) [reply jetstream/core/proto/jetstream.proto:30]: Sounds good!

**Review by FanhaiLu1 — APPROVED** [review]
LGTM

**Pull request merged**